en/lessons/basics/enum.md

```diff
@@ -143,6 +143,15 @@ Enum.max([], fn -> :bar end)
 :bar
 ```
 
+### filter
+
+`filter/2` returns collection's elements that match the condition from the given function.
```

Comment:
How about something like:

> The `filter/2` function enables us to filter the collection to include only those elements that evaluate to `true` using the provided function.

Reply:
Sounds clearer Sean! 😀 Will update then!

Reply:
Thanks, @wilbertliu!
Should we bump the minor version of the lesson?

Reply:
Hello @nscyclone, I'm not sure I understand what you intended to ask. Could you please elaborate more? 🙏🏻

Reply:
@wilbertliu, there is a variable called [`version`](https://github.com/elixirschool/elixirschool/pull/1349/files#diff-dfa183a1580b77baec65b2a1f09e3948L2) in the [front matter](https://jekyllrb.com/docs/frontmatter/) of every lesson/translation that helps to keep an eye on differences between original lessons and their translations. Usually when we introduce changes to an existing file we also increase `major`, `minor` or `patch` version. You can find a little more information about this in [`README`](https://github.com/elixirschool/elixirschool#translated-lesson).

In my opinion, in this case update of the `minor` version number would fit well to reflect the incoming changes, WDYT?

Reply:
@nscyclone Ah, I see. And yes, this should be `minor`. Who is in charge to change that though? Me? Thanks a lot for explaining! 🙂

Reply:
@wilbertliu Sorry for a late reply! Would be great if you'd take care of that on your own. Thanks in advance!

Reply:
@nscyclone No problem mate! I think I'm done? 😊